src/point.rs

```diff
@@ -943,6 +1118,8 @@ impl<T: ApproxEq<T>, U> ApproxEq<Point3D<T, U>> for Point3D<T, U> {
     }
 }
 
+//-----------------------------------------------------------------------------
```

Comment:
Please don't add these.

Reply:
Okay, but would you mind an additional blank line? There are a lot of trait impls and it is easy to get confused in them (which is what happened, judging by a bit chaotic implementation before that PR).

With visual separation, it is much better to see what impls is exist (just fold all in VS Code)

Reply:
> Okay, but would you mind an additional blank line?

:+1:

Reply:
I have now looked at the comment in the context of the entire file. Do you consider to delete only this line specifically, or all visual separators, which I added?

Reply:
All of the separators, please.